daprdocs/content/en/developing-applications/building-blocks/distributed-lock/distributed-lock-api-overview.md

```diff
@@ -0,0 +1,43 @@
+---
+type: docs
+title: "Distributed lock overview"
+linkTitle: "Overview"
+weight: 1000
+description: "Overview of the distributed lock API building block"
+---
+
+## Introduction
+Locks are used to provide mutually exclusive access to a resource. For example, you can use a lock to:
+
+- Provide exclusive access to a database row, table, or an entire database
+- Lock reading messages from a queue in a sequential manner
+
+Any resource that is shared where updates occur can be the target for a lock. Locks are usually used on operations that mutate state, not on reads.
+
+Each lock has a name. The application determines the resources that the named lock accesses. Typically, multiple instances of the same application (multiple applications with the same app ID) use this named lock to exclusively access the resource and perform updates. 
```

Comment:
Remove `(multiple applications with the same app ID)`

Reply:
Remove `(multiple applications with the same app ID)`

Reply:
Fixed